template/user.html

```diff
@@ -0,0 +1,29 @@
+<link rel="import" href="../bower_components/app-storage/app-localstorage/app-localstorage-document.html">
+
+<dom-module id="papan-user-data">
+  <template>
+    <app-localstorage-document
+      key="papan-user-data"
+      data="{{user}}">
+    </app-localstorage-document>
+  </template>
+  <script>
+    Polymer({
+      is: 'papan-user-data',
+
+      properties: {
+        user: {
+          type: Object,
+          value: function() {
```

Comment:
https://www.polymer-project.org/1.0/docs/devguide/properties

At least according to the doc, value can be one of the following: `boolean`, `string`, `number`, or `function`.